src/test/kotlin/io/libp2p/pubsub/gossip/GossipRpcPartsQueueTest.kt

```diff
@@ -0,0 +1,262 @@
+package io.libp2p.pubsub.gossip
+
+import io.libp2p.core.PeerId
+import io.libp2p.etc.types.toProtobuf
+import io.libp2p.etc.types.toWBytes
+import io.libp2p.pubsub.gossip.builders.GossipParamsBuilder
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import pubsub.pb.Rpc
+import java.util.Collections
+import java.util.stream.Stream
+
+class GossipRpcPartsQueueTest {
+
+    class TestGossipQueue(params: GossipParams) : DefaultGossipRpcPartsQueue(params) {
+
+        fun shuffleParts() {
+            Collections.shuffle(parts)
```

Comment:
IntelliJ would prefer this as:
```suggestion
            parts.shuffle()
```

Reply:
Right. Fixed. 
Thought it creates a new shuffled instance